composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "pantheon/terminus-plugin-example",
+    "description": "An example Terminus command",
+    "type": "terminus-plugin",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": { "Pantheon\\TerminusHello": "src" }
+    },
+    "extra": {
+        "terminus": {
+            "compatible-version": "1.*"
```

Comment:
> I don't think anybody would think that 1.1.* would also include 1.2. Maybe I'm wrong.

No, you are not wrong, you are absolutely right. People write 1.1.* before 1.2 exists; then, when 1.2 comes out, they are sad, or they keep updating their composer.json every minor release. While this is better than simply using `>=1.1`, simply using `^1.1` is immensely preferable.

Some prefer longhand to `^` and `~` because folks are always confusing those two operations. I think that failing to add the end version range is a comparably rare mistake, but if you're okay with `^`, I think that is a fine compromise.